packages/cli/src/zosfiles/compare/CompareBaseHelper.ts

```diff
@@ -146,11 +146,11 @@ export class CompareBaseHelper {
                 });
             }
             // reading local file as buffer
-            lfContentBuf = fs.readFileSync(localFileHandle);
+            lfContentStr = fs.readFileSync(localFileHandle, "utf-8");
         } finally {
             if (localFileHandle != null) fs.closeSync(localFileHandle);
         }
-        return lfContentBuf;
+        return Buffer.from(lfContentStr);
```

Comment:
Can you please restore the original implementation of this function?
```
let lfContentStr: Buffer;
lfContentBuf = fs.readFileSync(localFileHandle);
return lfContentBuf;
```

It should behave the same but more efficiently. Without an encoding specified, the `readFileSync` method returns a buffer, so I don't think its necessary to load the file as a string and then convert it to a buffer.